cmd/snap/cmd_debug_timings.go

```diff
@@ -42,6 +43,7 @@ func init() {
 		func() flags.Commander {
 			return &cmdChangeTimings{}
 		}, changeIDMixinOptDesc.also(map[string]string{
+			"ensure": i18n.G("Show timings for a change related to the given Ensure"),
```

Comment:
what we are looking for are really subpart of Ensure and changes triggered from those,  so maybe s/Ensure/Ensure activity/ ?